test/run

```diff
@@ -10,17 +10,21 @@
 test -n $IMAGE_NAME \
   -a -n $VERSION
 
-test_dir="$(readlink -zf $(dirname "${BASH_SOURCE[0]}"))"
-image_dir=$(readlink -zf ${test_dir}/..)
+test_dir="$(readlink -zf $(dirname ${BASH_SOURCE[0]}))"
+image_dir="$(readlink -zf ${test_dir}/..)"
 cid_file=$(mktemp -u --suffix=.cid)
 
 # Since we built the candidate image locally, we don't want S2I attempt to pull
 # it from Docker hub
-s2i_args="--pull-policy=never "
+s2i_args="--force-pull=false "
```

Comment:
Important to leave in since latest source-to-image does not make the flag available anymore.